### ISSUE TITLE
Flase positive error for http status phrase

### ISSUE THREAD

**Issue by c0r0n3r** [PyCQA/pylint#2950]
### Steps to reproduce

Run pylint on the following code.

```python
"""Test file for HTTP status phrase false positive error reproduction"""

import http

print(http.HTTPStatus.NOT_IMPLEMENTED.phrase)  # pylint: disable=no-member
```

### Current behavior

```
************* Module test
test.py:5:6: E1101: Instance of 'NOT_IMPLEMENTED' has no 'phrase' member (no-member)

----------------------------------------------------------------------
Your code has been rated at -15.00/10 (previous run: -15.00/10, +0.00)
```
### Expected behavior

```
----------------------------------------------------------------------
Your code has been rated at 10.00/10 (previous run: -20.00/10, +30.00)
```

### pylint --version output

```
pylint 2.3.1
astroid 2.2.5
Python 3.7.3 (default, Mar 27 2019, 13:41:07) 
[GCC 8.3.1 20190223 (Red Hat 8.3.1-2)]
```

**Comment by PCManticore** [comment]
Thanks, I can reproduce the issue.

**Comment by mcauto** [comment]
I have similar case on pylint v2.4.1

E1101: Instance of '_HTTPStatus' has no 'name' member (no-member)

python enum.Enum has 'name' member.
